codecov.yml

```diff
@@ -18,7 +18,7 @@ coverage:
   status:
     patch:
       default:
-        target: auto
+        target: 60%
```

Comment:
you can mark this as 70 after this PR goes in  https://github.com/antrea-io/theia/pull/230